app/models/rule.rb

```diff
@@ -3,6 +3,22 @@
 # Stores information about rules, such as which profiles can it be
 # found in, what hosts are associated with it, etceter
 class Rule < ApplicationRecord
+  SORTED_SEVERITIES = Arel.sql(
+    <<-SQL
+    CASE WHEN severity = 'high' THEN 3
+         WHEN severity = 'medium' THEN 2
+         WHEN severity = 'low' THEN 1
+         ELSE 0
```

Comment:
Have you tried what would happen if `severity IS NULL`?

Reply:
:thinking: what should happen when `severity IS NULL`?

Reply:
Same as unknown IMO

Reply:
Confirmed. Sorry, I'm being cautious when I see a NULL in postgresql.

Reply:
You had a valid point, no worries :+1: